modules/lvms-reference-file.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// storage/logical-volume-manager-storage.adoc
+
+:_content-type: REFERENCE
+[id="lvms-reference-file_{context}"]
+= Reference
```

Comment:
Mentioned in the add'l resources links to this section, but a longer title might help make this more meaningful in links and search (btw I really like the inclusion of this module, gives me ideas for some of my YAML-heavy content :nerd_face:)